src/System.Security.Cryptography.Primitives/src/System/Security/Cryptography/CryptographicUnexpectedOperationException.cs

```diff
@@ -0,0 +1,31 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Globalization;
+
+namespace System.Security.Cryptography
+{
+    public class CryptographicUnexpectedOperationException : CryptographicException
+    {
+        public CryptographicUnexpectedOperationException()
+            : base(SR.Arg_CryptographyException)
+        {
+        }
```

Comment:
Possibly , but then we should also change CryptographicException as well as that doesn't set CORSEC_E_CRYPTO 

        internal const int CORSEC_E_CRYPTO = unchecked((int)0x80131430);
        internal const int CORSEC_E_CRYPTO_UNEX_OPER = unchecked((int)0x80131431);